server/tests/api_v4/authenticated/test_shamir_recovery_setup.py

```diff
@@ -108,7 +108,7 @@ async def test_authenticated_shamir_recovery_setup_shamir_setup_already_exists(
         [share.dump_and_sign(coolorg.alice.signing_key)],
     )
     rep = await coolorg.alice.shamir_recovery_setup(setup)
-    assert rep == authenticated_cmds.v4.shamir_recovery_setup.RepShamirSetupAlreadyExists()
+    assert isinstance(rep, authenticated_cmds.v4.shamir_recovery_setup.RepShamirSetupAlreadyExists)
```

Comment:
You should probably check the timestamp too. Something like:
```suggestion
    assert rep == authenticated_cmds.v4.shamir_recovery_setup.RepShamirSetupAlreadyExists(
        last_shamir_certificate_timestamp=dt
    )
```

Reply:
dt is the timestamp of the new certificate we're trying to add, it's not the timestamp of the already present setup.

For now `setup_shamir_for_coolorg` return nothing, but there is a commit somewhere when it returns the brief and share certificates to we will be able to use that.

Reply:
OK, in that case the last two options seem fine. Just follow what you think is the shortest path :)

Reply:
I'll go for the third option